memory-viz/src/cli.ts

```diff
@@ -77,45 +80,41 @@ program
     .option("--no-interactive", "disable hover interactivity for object IDs");
 
 program.parse();
-const filePath = program.processedArgs[0];
+const filePath: string | undefined = program.processedArgs[0];
 const options = program.opts();
 
 if (filePath) {
-    const fileContent = fs.readFileSync(filePath, "utf8");
-
-    let jsonContent;
-    try {
-        jsonContent = JSON.parse(fileContent);
-    } catch (err) {
-        console.error(`Error: Invalid JSON\n${err.message}.`);
-        process.exit(1);
-    }
-
-    runMemoryViz(jsonContent);
+    runMemoryViz(filePath);
 } else {
     json(process.stdin)
         .then((jsonContent) => {
-            runMemoryViz(jsonContent);
+            runMemoryViz(jsonContent as DrawnEntity[] | DrawnEntity[][]);
         })
-        .catch((err) => {
+        .catch((err: any) => {
             console.error(`Error: ${err.message}.`);
             process.exit(1);
         });
 }
 
-function runMemoryViz(jsonContent) {
-    let m;
+function runMemoryViz(
+    jsonContent: string | DrawnEntity[] | DrawnEntity[][]
+): void {
+    let m: any;
     try {
-        m = draw(jsonContent, true, {
+        m = draw(jsonContent as any, true, {
```

Comment:
Similar to my above comment, I'm a bit surprised `as any` is here. What is the error you're seeing with this?